lib/core/src/lib/common/utils/date-fns-utils.spec.ts

```diff
@@ -53,27 +52,18 @@ describe('DateFnsUtils', () => {
     });
 
     it('should parse datetime', () => {
-        const parsed = DateFnsUtils.parseDate(
-            '09-02-9999 09:10 AM',
-            'dd-MM-yyyy hh:mm aa'
-        );
+        const parsed = DateFnsUtils.parseDate('09-02-9999 09:10 AM', 'dd-MM-yyyy hh:mm aa');
```

Comment:
all these format changes has been done automatically by linter when I did the commit